web_m2x_options/static/src/js/form.js

```diff
@@ -0,0 +1,388 @@
+/* Copyright 2016 0k.io,ACSONE SA/NV
+ *  * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_m2x_options.web_m2x_options', function (require) {
+    "use strict";
+
+    var core = require('web.core'),
+        data = require('web.data'),
+        Dialog = require('web.Dialog'),
+        view_dialogs = require('web.view_dialogs'),
+        relational_fields = require('web.relational_fields'),
+        rpc = require('web.rpc');
+
+    var _t = core._t,
+        FieldMany2ManyTags = relational_fields.FieldMany2ManyTags,
+        FieldMany2One = relational_fields.FieldMany2One,
+        FormFieldMany2ManyTags = relational_fields.FormFieldMany2ManyTags;
+
+    var web_m2x_options = rpc.query({
+        model: "ir.config_parameter",
+        method: 'get_web_m2x_options',
+    });
+
+    var M2ODialog = Dialog.extend({
```

Comment:
M2ODialog now completely copy pasted to odoo I think you can remove it completly
https://github.com/odoo/odoo/blob/08540d7853d497e326182e822b359cd6fcb05d59/addons/web/static/src/js/fields/relational_fields.js#L38

Reply:
Great; thanks. I removed it already but i'm still trying to find out what is travis complaining about. :(